specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualWan.json

```diff
@@ -3993,22 +3993,35 @@
     },
     "P2SVpnConnectionHealthRequest": {
       "properties": {
-        "p2sVpnConnectionIdsOrVpnUserNames": {
+        "p2sVpnConnectionHealthFilters": {
           "type": "array",
           "items": {
-            "type": "string"
+            "type": "p2sVpnConnectionHealthFilter"
           },
           "description": "List of P2S Vpn connection Ids of Vpn user names."
         },
         "outputBlobSasUrl": {
           "type": "string",
           "description": "The sas-url to download the P2S Vpn connection health detail."
         }
+      },      
+      "description": "List of P2S Vpn connection health details."
+    },
+    "p2sVpnConnectionHealthFilter": {
+      "properties": {
+        "p2sConnectionConfigurationId": {
+          "type": "string",
+          "description": "p2s connection configuration Id"
```

Comment:
Please start description with uppercase letter and end it with period to keep it withing Swagger guidelines